### PR TITLE
#122 added methods for removing component using custom clickable area

### PR DESCRIPTION
**Pull request by kkarolk**

### CODE REVIEW

File: bobcat/bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/parsys/AemParsys.java [hunk 1]
```diff
@@ -439,7 +464,7 @@ private By getComponentLocator(String componentCssClass) {
     return By.cssSelector(String.format(componentLocatorFormat, componentCssClass));
   }
 
-  private void clickDeleteInContextMenu(WebElement webElement) {
+  private void clickDeleteInContextMenuAndAcceptConfirmationWindow(WebElement webElement) {
```
Comment (mkrzyzanowski) [inline bobcat/bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/parsys/AemParsys.java]: `deleteComponent` would suffice, especially as it's a private method.

File: bobcat/bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/parsys/AemParsys.java [hunk 2]
```diff
@@ -462,4 +487,9 @@ private WebElement getClickableParsys() {
       return xPositionDiff + yPositionDiff <= parsysOffset;
     };
   }
+
+  private void waitForComponentRemoved(WebElement webElement) {
+    wait.withTimeout(Timeouts.SMALL).until(ExpectedConditions.stalenessOf(webElement));
```
Comment (mkrzyzanowski) [inline bobcat/bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/parsys/AemParsys.java]: I wonder if 5 sec is enough for heavier pages.

File: bobcat/bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/parsys/AemParsys.java [hunk 3]
```diff
@@ -462,4 +487,9 @@ private WebElement getClickableParsys() {
       return xPositionDiff + yPositionDiff <= parsysOffset;
     };
   }
+
+  private void waitForComponentRemoved(WebElement webElement) {
```
Comment (mmanski) [inline bobcat/bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/parsys/AemParsys.java]: waitForComponentRemoval or waitForComponentToBeRemoved to be more grammatically correct?

**Comment by mmanski** [comment]
👍

**Pull request merged**